earthly/rust/stdcfgs/deny.toml

```diff
@@ -268,7 +268,7 @@ allow-git = []
 
 [sources.allow-org]
 # 1 or more github.com organizations to allow git sources for
-#github = [""]
+github = ["input-output-hk"]
```

Comment:
I would prefer we don't do this, for the pallas fork especially.
The pallas fork should not be long lived, we should be upstreaming any changes we do ASAP.
If we DO use a github repo directly, I would also rather we specifically declare it and not a whole organization even if its us.